src/arch/runtime/event_queue/iocp.hpp

```diff
@@ -1,12 +1,15 @@
 #ifndef ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 #define ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 
+#include <stdint.h>
```

Comment:
That doesn't make it right. [`stdint.h` is deprecated](http://stackoverflow.com/questions/13642827/cstdint-vs-stdint-h).